xml/api-result-0.1.rng

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grammar xmlns="http://relaxng.org/ns/structure/1.0"
+         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+    <start>
+        <ref name="element-pacemaker-result"/>
+    </start>
+
+    <define name="element-pacemaker-result">
+        <element name="pacemaker_result">
+            <attribute name="api_version"> <text /> </attribute>
+            <attribute name="request"> <text /> </attribute>
+            <choice>
+                <element name="installed_fence_devices">
```

Comment:
Counterproposal to this movement is to keep `pacemaker-*.rng` schemas
where they are, and only move the particular constituents into `/cib/`
(relative to the parent, which is `xml` directory in tree,
`/usr/share/pacemaker` when deployed).

Non-negligible advantage of this is retaining full "data-at-rest API"
compatibility.  And one doesn't need to edit `pacemaker-*.rng` directly
anyway (it's auto-generated) so that the distinction makese sense also
semantically.